analysis/type.go

```diff
@@ -50,9 +51,12 @@ type Token struct {
 
 	// Position specifies the 1-based index of the token in the sequence of
 	// occurrences of its term in the field.
-	Position int       `json:"position"`
-	Type     TokenType `json:"type"`
-	KeyWord  bool      `json:"keyword"`
+	Position    int       `json:"position"`
+	Type        TokenType `json:"type"`
+	KeyWord     bool      `json:"keyword"`
```

Comment:
a small nit, keep the naming convention consistent like Keyword should be enough over here.